lib/core/src/Cardano/DB/Sqlite.hs

```diff
@@ -479,11 +480,18 @@ instance ToText DBLog where
             "Database "+|wid|+" is ready to be deleted"
         MsgWaitingForDatabase wid (Just count) ->
             "Waiting for "+|count|+" withDatabase "+|wid|+" call(s) to finish"
-        MsgRemovingInUse wid count ->
-            "Timed out waiting for "+|count|+" withDatabase "+|wid|+" call(s) to finish. " <>
-            "Attempting to remove the database anyway."
+        MsgRemovingInUse wid count -> mconcat
```

Comment:
Why replace the combinators from a _string formatting library_ with `mconcat`? 
We should be doing the reverse.
In any case, it's not related to the issue.

Reply:
> Why replace the combinators from a _string formatting library_ with `mconcat`?

Thanks for spotting that: removing the combinators was unintentional.

I've removed this commit.